app/move/app/new/controllers/move-details.js

```diff
@@ -66,21 +66,30 @@ class MoveDetailsController extends CreateBaseController {
       return true
     }
 
-    const move = req.getMove()
-    const locationId = move.from_location?.id || move.from_location
-    const location = await locationService.findById(req, locationId, true)
+    const location = await this.getMoveLocation(req)
 
-    if (
-      location.suppliers
-        .map(supplier => supplier.key)
-        .some(supplier =>
-          FEATURE_FLAGS.AP_DISABLED_SUPPLIERS.includes(supplier)
-        )
-    ) {
+    return !location.suppliers
+      .map(supplier => supplier.key)
+      .some(supplier => FEATURE_FLAGS.AP_DISABLED_SUPPLIERS.includes(supplier))
+  }
+
+  async extraditionMovesAllowed(req) {
+    if (!FEATURE_FLAGS.EXTRADITION_MOVES) {
+      return false
+    }
+
+    if (!this.isFromGivenLocationType(req, 'prison')) {
       return false
     }
 
-    return true
+    const location = await this.getMoveLocation(req)
```

Comment:
It calls `await this.getMoveLocation(req)` from `extraditionMovesAllowed` and `apMovesAllowed` so it would be called twice I think - not the end of the world but maybe could be done more efficiently?

Reply:
Both suppliers now support AP moves and the feature flag for that was redundant, so have removed the code that performs that call there